### PR TITLE
Add raw.var requirements

### PR DESCRIPTION
**Pull request by pablo-gar**
Adds requirements to `raw.var` which must be identical to `var`.

### CODE REVIEW

File: schema/2.0.0/corpora_schema.md [hunk 1]
```diff
@@ -26,16 +26,17 @@ This schema supports multiple assay types. Each assay takes the form of one or m
 The schema additionally describes how the dataset, genes, and cells are annotated to describe the biological and technical characteristics of the data.
 
 This document is organized by:
+
 * [General requirements](#general-requirements)
 * [`X` (Matrix layers)](#x-matrix-layers), which describe the data required for different assays
 * [`obs` (Cell metadata)](#obs-cell-metadata), which describe each cell in the dataset
-* [`var` (Gene metadata)](#var-gene-metadata), which describe each gene in the dataset
+* [`var`/`raw.var` (Gene metadata)](#var-gene-metadata), which describe each gene in the dataset
```
Comment (brianraymor) [inline schema/2.0.0/corpora_schema.md:33]: The link is broken. Can you try:
`[`var` and `raw.var` (Gene Metadata)](#var-and-rawvar-gene-metadata])`

File: schema/2.0.0/corpora_schema.md [hunk 2]
```diff
@@ -905,6 +906,7 @@ schema v2.0.0 substantially *remodeled* schema v1.1.0:
 * var
   * Replaced HGNC gene **symbols** as `var.index` with ENSEMBL or ERCC spike-in **identifiers** 
   * Added `feature_id`, `feature_name`, and `feature_reference`
+  * Added requirements for `raw.var` which should be identical to `var`
```
Comment (brianraymor) [inline schema/2.0.0/corpora_schema.md:909]: To be precise, should->must

**Comment by pablo-gar** [comment]
@brianraymor your comments have addressed

**Review by brianraymor — APPROVED** [review]

File: schema/2.0.0/corpora_schema.md (hunk 2 above)
Comment (jychien) [reply schema/2.0.0/corpora_schema.md]: Is this still in discussion? Does var index need to be _identical_ to raw.var index or can it be a _subset of_ raw.var index?

**Pull request merged**